### ISSUE TITLE
Display indicator when the web UI is waiting on an API response

### ISSUE THREAD

**Issue by amdprophet** [sensu/sensu-go#2633]
## Expected Behavior
When using a feature (e.g. pagination) in the web UI that depends on an API response, display some sort of loading icon/toolbar/etc. that shows the user that the change is loading.

## Current Behavior
There are no loading indicators and pagination gives the illusion of not working when the API is taking a long time to respond.

**Comment by jamesdphillips** [comment]
Duplicate of sensu/web#18
